packages/raml-generator/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@commercetools-docs/raml-generator",
+  "version": "1.0.0",
+  "description": "Generates RAML docs compatible with @commercetools-docs/gatsby-transformer-raml",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "src/index.js",
```

Comment:
This should be removed. The package does not export any API or something to be used as a JS package